creator-node/src/ipfsClient.js

```diff
@@ -15,11 +19,102 @@ const ipfsLatest = ipfsClientLatest({ host: ipfsAddr, port: config.get('ipfsPort
 async function logIpfsPeerIds () {
   const identity = await ipfs.id()
   // Pretty print the JSON obj with no filter fn (e.g. filter by string or number) and spacing of size 2
-  logger.info(`Current IPFS Peer ID: ${JSON.stringify(identity, null, 2)}`)
+  genericLogger.info(`Current IPFS Peer ID: ${JSON.stringify(identity, null, 2)}`)
 
   // init latest version of ipfs
   const identityLatest = await ipfsLatest.id()
-  logger.info(`Current IPFS Peer ID (using latest version of ipfs client): ${JSON.stringify(identityLatest, null, 2)}`)
+  genericLogger.info(`Current IPFS Peer ID (using latest version of ipfs client): ${JSON.stringify(identityLatest, null, 2)}`)
+}
+
+/**
+ * Wrapper to ipfs.addToFs() -- This wrapper allows for ipfs calls to be made async or synchronous.
+ * @param {Object} ipfs ipfs library -- can be two different versions (see ipfsClient.js)
+ * @param {Buffer|Buffer[]} buffers
```

Comment:
api here is weird - its either a single buffer, or an array of objects of type { path: string, content: buffer }. can we update this comment / varname